Plugins/GRpluginStands.txt

```diff
@@ -1167,115 +1167,215 @@ USE:C
 CODE:E
 
 //STAND 313
-STAND:ENGM:313:N060.11.55.530:E011.04.34.750:20
+STAND:ENGM:313
+COORD:N060.11.56.214:E011.04.37.733
+COORD:N060.11.55.301:E011.04.37.898
+COORD:N060.11.55.106:E011.04.33.037
+COORD:N060.11.56.029:E011.04.32.926
+COORD:N060.11.56.214:E011.04.37.733
 USE:BP
 CODE:B
 PRIORITY:+1
 
 //STAND 314
-STAND:ENGM:314:N060.11.56.530:E011.04.34.630:20
+STAND:ENGM:314
+COORD:N060.11.57.215:E011.04.37.547
+COORD:N060.11.56.303:E011.04.37.712
+COORD:N060.11.56.107:E011.04.32.851
+COORD:N060.11.57.030:E011.04.32.741
+COORD:N060.11.57.215:E011.04.37.547
 USE:BP
 CODE:B
 PRIORITY:+1
 
 //STAND 315
-STAND:ENGM:315:N060.11.57.530:E011.04.34.510:20
+STAND:ENGM:315
+COORD:N060.11.58.226:E011.04.37.388
+COORD:N060.11.57.314:E011.04.37.554
+COORD:N060.11.57.118:E011.04.32.692
+COORD:N060.11.58.041:E011.04.32.582
+COORD:N060.11.58.226:E011.04.37.388
 USE:BP
 CODE:B
 PRIORITY:+1
 
 //STAND 316
-STAND:ENGM:316:N060.11.58.530:E011.04.34.380:20
+STAND:ENGM:316
+COORD:N060.11.59.218:E011.04.37.222
+COORD:N060.11.58.306:E011.04.37.388
+COORD:N060.11.58.110:E011.04.32.526
+COORD:N060.11.59.034:E011.04.32.416
+COORD:N060.11.59.218:E011.04.37.222
 USE:BP
 CODE:B
 PRIORITY:+1
 
 //STAND 317
-STAND:ENGM:317:N060.11.59.520:E011.04.34.260:20
+STAND:ENGM:317
+COORD:N060.12.00.249:E011.04.37.071
+COORD:N060.11.59.336:E011.04.37.236
+COORD:N060.11.59.141:E011.04.32.375
+COORD:N060.12.00.064:E011.04.32.264
+COORD:N060.12.00.249:E011.04.37.071
 USE:BP
 CODE:B
 PRIORITY:+1
 
 //STAND 318
-STAND:ENGM:318:N060.12.00.640:E011.04.34.080:20
+STAND:ENGM:318
+COORD:N060.12.01.349:E011.04.36.892
+COORD:N060.12.00.437:E011.04.37.058
+COORD:N060.12.00.241:E011.04.32.196
+COORD:N060.12.01.164:E011.04.32.086
+COORD:N060.12.01.349:E011.04.36.892
 USE:BP
 CODE:C
 PRIORITY:+1
 
 //STAND 319
-STAND:ENGM:319:N060.12.02.940:E011.04.33.870:20
+STAND:ENGM:319
+COORD:N060.12.03.017:E011.04.36.683
+COORD:N060.12.02.105:E011.04.36.849
+COORD:N060.12.01.909:E011.04.31.987
+COORD:N060.12.02.832:E011.04.31.877
+COORD:N060.12.03.017:E011.04.36.683
```

Comment:
Need to test the position of this polygon somewhat. Either it's a bit off or the label should be moved.